### PR TITLE
Hello Microverse

### PR DESCRIPTION
**Pull request by Kumilachew-g**
I have completed the following tasks
  1. Configured linters for CSS and HTML.
  2. Added project readme.
  3. Added index.html and Styling.
  4. Added gitignore file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,46 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
```
Comment (Ismailco) [inline README.md:1]: - [ ] The Pull Request title should reflect the changes in the PR, kindly consider changing it to more descriptive title

Comment (Ismailco) [inline README.md:1]: - [ ] [OPTIONAL] You should have 2 files committed: a .html and .css file, kindly consider removing other files in the PR

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (Ismailco) [inline .gitignore]: - [ ] The `.gitignore` file should contain `test.md`, in addition to the `node)modules` folder, kindly implement this so the test.md is ignored by git

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @Kumilachew-g,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — APPROVED** [review]
Hi @Kumilachew-g,

## APPROVED 🟢

✔️ Passing linters

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**